disperser/apiserver/server_test.go

```diff
@@ -683,6 +685,22 @@ func newTestServer(transactor core.Writer, testName string) *apiserver.Dispersal
 		panic("failed to make initial query to the on-chain state")
 	}
 
+	mockState.On("GetPricePerSymbol").Return(uint32(encoding.BYTES_PER_SYMBOL), nil)
```

Comment:
ha uhhh this is because the previous tests was made on the basis of charging per byte instead of per symbol, and I was lazy to redo the calculation after the blob size conversion, so I just scaled the price 😅